keyboards/skdevices/adler/rules.mk

```diff
@@ -0,0 +1,2 @@
+I2C_DRIVER_REQUIRED = yes
+PROGRAMMABLE_BUTTON_ENABLE = yes
```

Comment:
```suggestion
```

i2c driver required setting isn't required.  

Also, the programmable button feature isn't meant for macros or custom functionality. It's a host side thing. 
<https://docs.qmk.fm/features/programmable_button>